src/js/libcs/CSNumber.js

```diff
@@ -774,180 +774,17 @@ CSNumber._helper.solveCubicHelper = function(a, b, c, d) {
     ]);
 };
 
+CSNumber._helper.getRangeRand = function(min, max) {
+    return Math.random() * (max - min + 1) + min;
```

Comment:
Why do we need this `+ 1` here? This function might return numbers greater than `max`.